packages/legend-studio/src/stores/StudioPlugin.ts

```diff
@@ -141,6 +143,11 @@ export type ElementProjectExplorerDnDTypeGetter = (
   element: PackageableElement,
 ) => string | undefined;
 
+export type ExtraFileGenerationtButton = (
```

Comment:
Please change this to `FileGenerationResultViewerAction`

Reply:
Fixed!